### ISSUE TITLE
Thumbnail documentation/api is confusing

### ISSUE THREAD

**Issue by Mike-Bell** [microsoftgraph/microsoft-graph-docs-contrib#8391]
Several related things here. http://graph.microsoft.io/docs/api-reference/v1.0/api/thumbnailset_update lists several routes for updating a thumbnailset:

PATCH /drive/root/thumbnails/`<id>`
PATCH /drive/items/`<id>`/thumbnails/`<id>`
PATCH /drives/`<id>`/root/thumbnails/`<id>`

Two of these routes have... two ID parameters? Are those the same ID? Is one the item ID for the item that contains the thumbnails and one the ID of the thumbnailset itself? Are they the same ID? Why does the first route only have one ID param? Can I use an item path rather than ID? It's also not at all clear what's required for the body of the PATCH. What is actually necessary to pass in to make a custom thumbnailset work?

The Get thumbnailset page (http://graph.microsoft.io/docs/api-reference/v1.0/api/thumbnailset_get) links to this page http://graph.microsoft.io/docs/api-reference/v1.0/api/item_list_thumbnails for more info. But some of the info in here seems to conflict with what was listed on http://graph.microsoft.io/docs/api-reference/v1.0/api/thumbnailset_update. This page seems to suggest that you can do a PUT to upload a custom thumbnail, as opposed to the Update page, which indicated that a PATCH is necessary. Or is there a difference between updating a thumbnailset and updating a thumbnail? I'm not sure. The Update page also indicates that custom thumbnails don't work on OD4B which http://graph.microsoft.io/docs/api-reference/v1.0/api/thumbnailset_update did not mention. From my experimenting, doing a PATCH against a thumbnailset on OD4B returns a 200, but you don't get those thumbnails back when you request them. Is this the intended behavior? Did the request silently fail? 

I can grab thumbnails for an image (hooray...) but videos don't seem to get thumbs on OD4B and I can't seem to come up with a way to upload a custom one.

TLDR, working with thumbnails seems very confusing, and possibly non-functional.
